config/validation.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+	"github.com/contiv/errored"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+
+	gojson "github.com/xeipuuv/gojsonschema"
+)
+
+// Combines array of errors into a single error
+func combineErrors(resultErrors []gojson.ResultError) error {
+	var errors []string
+	for _, err := range resultErrors {
+		errors = append(errors, fmt.Sprintf("%s\n", err))
+	}
+	return errored.New(strings.Join(errors, "\n"))
+}
+
+// ValidateRuntime validates the given runtime against its defined schema
+func ValidateRuntime(cfg *RuntimeOptions) error {
+	schema := gojson.NewStringLoader(RuntimeSchema)
+	doc := gojson.NewGoLoader(cfg)
+
+	if result, _ := gojson.Validate(schema, doc); !result.Valid() {
+		return combineErrors(result.Errors())
+	}
+	log.Infof("Runtime config validation passed")
+	return nil
+}
+
+// ValidatePolicy validates the given policy against its defined schema
+func ValidatePolicy(cfg *Policy) error {
```

Comment:
I can do it. But I wanted the schema validation to be in separate function, just not to make Validate functions noisy. Please input your thoughts.